scripts/config.ts

```diff
@@ -32,7 +32,7 @@ DEPOSIT_CONTRACT_ADDRESS: 0x4242424242424242424242424242424242424242
 }
 
 function writeGethGenesisConfig(argv: any) {
-   const gethConfig =  `
```

Comment:
I think we shouldn't reformat the files if we're working on forks. It will make conflict resolution more troublesome than it already is when we sync changes from upstream.

Reply:
Agreed, I can remove the formatting